api/src/main/java/io/opentelemetry/api/trace/propagation/HttpTraceContext.java

```diff
@@ -33,7 +33,7 @@ public static HttpTraceContext getInstance() {
   }
 
   @Override
-  public List<String> fields() {
+  public Collection<String> fields() {
     return W3CTraceContextPropagator.getInstance().fields();
```

Comment:
It seems the `W3CTraceContextPropagator`'s implementation should also be changed.

Reply:
Additionally, when doing a CMD + SHFT + F in IntelliJ for `fields()`.. It also highlights the following `fields()` methods in the propagators.

![image](https://user-images.githubusercontent.com/29541485/100616216-ad01c600-3310-11eb-8c57-1bfd7185aa3c.png)

Do you think we should also change these implementations? As although they are `List` which is a subtype of `Collection`, it may be best to be explicit with the types for clarify and consistency.

Reply:
Yes, let's change them all for consistency. good catch.

Reply:
Pushed changes for all implementations also